src/WebJobs.Script.WebHost/Management/IFunctionsSyncManager.cs

```diff
@@ -2,11 +2,16 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Scale;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
 {
     public interface IFunctionsSyncManager
     {
+        IExtensionsOptionProvider ExtensionsOptionProvider { set; }
```

Comment:
We don't want these properties in the public interface

Reply:
Sure. I'll remove it from the interface.

Reply:
Removed.